cpu/stm32_common/periph/gpio.c

```diff
@@ -46,6 +46,16 @@
 static gpio_isr_ctx_t isr_ctx[EXTI_NUMOF];
 #endif /* MODULE_PERIPH_GPIO_IRQ */
 
+/* L412KB specifics */
+#ifdef CPU_MODEL_STM32L412KB
```

Comment:
```suggestion
#if !defined(IMR) && defined(IMR1)
```

This way things should just work™ on future MCUs.

Reply:
Thanks!

Reply:
Oops. sorry I did not realize this was a member of a struct.

```suggestion
#if !defined(EXTI_IMR_IM) && defined(EXTI_IMR1_IM)
```

should work then.

Reply:
I guess the pins connected to IMR2 need separate handling then.